levels/branches/checkout-commit

```diff
@@ -3,7 +3,7 @@ cards = checkout commit-auto
 
 [description]
 
-The yellow boxes are frozen points in time, we call them "commits"! You can travel between them using the "checkout" card! (Try it!)
+The yellow boxes are frozen points in time, we call them 'commits'! You can travel between them using the "checkout" card! (Try it!)
```

Comment:
why is only a pair of double quotes changed to singe ones?

Reply:
In .po files, the msgid is wrap in double quotes and the string must match perfectly

Reply:
In this case i have changed manually that string for test but in the script i have add code to replace the double quote in single quote